### PR TITLE
added weight decay

### PR DESCRIPTION
**Pull request by ashish-sasha**
In the `Implementation details` of the paper it is mentioned that `The
initial learning rate is set to 0.1, weight decay to 0.0005, dampening to 0, momentum to 0.9
and minibatch size to 128`. Since keras has no global weight decay; as mentioned here `https://github.com/keras-team/keras/issues/2717`, it is recommended to add weight decay using `Regularizers` on every applicable layer.

### CODE REVIEW

**Comment by ashish-sasha** [comment]
closes #15

**Comment by titu1994** [comment]
Thanks for the PR !
